docs/usage/updates-strategy.md

```diff
@@ -35,7 +35,7 @@ In short, it consists of two operations:
  * unlock: after rebooting, any reboot slot owned by the node must be unlocked (and confirmed) by the lock-manager before proceeding further.
 
 This protocol is not coupled to any specific backend, and can be implemented on top of any suitable database.
-As an example, [airlock] is a free-software project which implements such protocol on top of [etcd3].
+As an example, [airlock](https://github.com/coreos/airlock) is a free-software project which implements such protocol on top of [etcd3](https://etcd.io/).
```

Comment:
It looks like my brain had an hiccup here. There are references at the bottom of this section, but I forgot to link them here (and the `airlock` one is plainly missing).

Can you please align with the existing bottom references?

Reply:
Today I learned a new way to make links in Markdown!

Reply:
A win-win PR! ;)